metabase/metabase
# Browse metrics page

Closes https://github.com/metabase/metabase/issues/47011

- Adds a simple browse metrics page
- Fixes an issue where markdown descriptions on the models page were not rendered as markdown
- This PR was copying a lot of code from the `/browse/models` page, and that code was getting a lot of review comments, so I applied them to the original code too.

## To verify

- Make sure you have some metrics created
- Click the `Metrics` title in the sidebar
- You should be taken to the `/browse/metrics` page, where you see all your metrics
- Play around with sorting


<img width="1474" alt="Screenshot 2024-08-27 at 16 35 38" src="https://github.com/user-attachments/assets/9b4314d9-4d1c-441d-a3ff-789dd376c1dd">
